pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/worker/WorkerConfig.java

```diff
@@ -99,6 +99,17 @@
         doc = "The port for serving worker https requests"
     )
     private Integer workerPortTls;
+    @FieldContext(
+            category = CATEGORY_WORKER,
+            doc = "Whether the '/metrics' endpoint requires authentication. Defaults to false."
```

Comment:
does it default to false or true? The doc says one and the default config says other